nomos-http-api/src/http/backend/axum.rs

```diff
@@ -0,0 +1,48 @@
+use overwatch_rs::overwatch::handle::OverwatchHandle;
+use std::{net::SocketAddr, sync::Arc};
+
+use axum::{Router, Server};
+use utoipa::OpenApi;
+use utoipa_swagger_ui::SwaggerUi;
+
+use crate::Backend;
+
+#[derive(Clone)]
+pub struct AxumBackendSettings {
+    pub da: OverwatchHandle,
```

Comment:
what's this field for?

Reply:
Ah, this is a left-over when splitting the PR.